src/core/fonts.js

```diff
@@ -1305,6 +1308,15 @@ var Font = (function FontClosure() {
           data[property] = value;
         }
       }
+      if (extraProperties) {
```

Comment:
Instead of repeating this loop, would it be possible to concatenate `EXPORT_DATA_PROPERTIES` with `EXPORT_DATA_EXTRA_PROPERTIES` above in case `extraProperties = true` instead?

Reply:
It's obviously possible to do that, but I wanted to avoid doing extra work/allocations in the "normal" case. The only solution satisfying all these criteria, which I'm able to come up with, is perhaps slightly ugly; please see following the diff:
```diff
diff --git a/src/core/fonts.js b/src/core/fonts.js
index f5b6e09c..241e50da 100644
--- a/src/core/fonts.js
+++ b/src/core/fonts.js
@@ -1299,24 +1299,19 @@ var Font = (function FontClosure() {
     },
 
     exportData(extraProperties = false) {
+      const exportDataProperties = extraProperties
+        ? [...EXPORT_DATA_PROPERTIES, ...EXPORT_DATA_EXTRA_PROPERTIES]
+        : EXPORT_DATA_PROPERTIES;
+
       const data = Object.create(null);
       let property, value;
-      for (property of EXPORT_DATA_PROPERTIES) {
+      for (property of exportDataProperties) {
         value = this[property];
         // Ignore properties that haven't been explicitly set.
         if (value !== undefined) {
           data[property] = value;
         }
       }
-      if (extraProperties) {
-        for (property of EXPORT_DATA_EXTRA_PROPERTIES) {
-          value = this[property];
-          // Ignore properties that haven't been explicitly set.
-          if (value !== undefined) {
-            data[property] = value;
-          }
-        }
-      }
       return data;
     },
```
Do you think that would be a better approach overall?